src/mixins/filing-template-mixin.ts

```diff
@@ -267,8 +252,6 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
     }
 
     // restore Office Addresses
-    // *** TODO: verify whether we need to assign fallback
-    // *** also fix IAs and registrations the same way?
```

Comment:
I've looked into this, and I don't see the benefit of assigning a fallback. Please correct me if I'm wrong.

Reply:
What I was thinking here is that, if the draft filing doesn't contain office addresses, we'd assign a safe fallback value (eg, empty object?). Similarly to what we do if don't have parties or shareClasses to restore.

However, theoretically the store has safe initial values for offices / parties / shareClasses, so the correct code is "set the value in the store only if the draft contains the value", which is what lines 272-273 already do.

But I'm not certain that our code is consistently doing the right thing. For example:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/149972f4-cbb7-4a87-8a84-eeffb568fe81)

You'd think the fallback to an empty array wouldn't be needed, but in fact parties is initialized to null in the store. It should probably be initialized to an empty array, but I don't know if that will break other filings.

So, to move forward with this, either we try to fix the code now and test well, or we ignore this for now and test well (this is safe but won't resolve the code issues). 🤷‍♂️

Reply:
Makes a lot of sense. Sev, I'll be tackling and looking into this more in next commit.